### ISSUE TITLE
Nubiles Match Issues

### ISSUE THREAD

**Issue by HWatBobby** [PAhelper/PhoenixAdult.bundle#290]
Found an issue pulling data from Nubiles-Porn sites. Seeing issue with Bratty Sis, Moms Teach Sex, My Family Pies, PetiteHD, Princess Cum, and Step Siblings Caught. Didn't see any issues regarding this within the past 3 weeks. This issue occurs with every Nubiles scene I search for (old & new).

One video with an issue is Bratty Sis - Caught Sneaking Out @ https://brattysis.com/video/watch/46308

When attempting to match I search using the term Bratty Sis 46308. I can see the correct url is provided in the log.

File name is Bratty Sis - 2017-05-12 - 46308 - S01E01 - Tiffany Watson - Caught Sneaking Out.mp4

Also tried renaming the file to Bratty Sis - 46308 and got the same results. 

**Here is the log data I received for the selected scene:**

```
2019-07-17 20:34:10,239 (14ecf024a700) :  DEBUG (runtime:88) - Sending packed state data (1016 bytes)
2019-07-17 20:34:10,239 (14ecf024a700) :  DEBUG (runtime:924) - Response: [200] str, 320 bytes
2019-07-17 20:34:23,472 (14ecf024a700) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmVuczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGUzCnM1CjI2MTk4czIKaWRzMTYKQnJhdHR5IFNpcyA0NjMwOHM0Cm5hbWVuczQKeWVhcnIwCg__
2019-07-17 20:34:23,472 (14ecf024a700) :  DEBUG (runtime:49) - Received packed state data (984 bytes)
2019-07-17 20:34:23,473 (14ecf024a700) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmVuczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGUzCnM1CjI2MTk4czIKaWRzMTYKQnJhdHR5IFNpcyA0NjMwOHM0Cm5hbWVuczQKeWVhcnIwCg__
2019-07-17 20:34:23,474 (14ecf024a700) :  INFO (agentkit:957) - Searching for matches for {'year': None, 'id': '26198', 'name': 'Bratty Sis 46308'}
2019-07-17 20:34:23,474 (14ecf024a700) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/library/metadata/26198/tree'
2019-07-17 20:34:23,478 (14ecf024a700) :  INFO (__init__:46) - *******MEDIA TITLE****** Bratty Sis 46308
2019-07-17 20:34:23,478 (14ecf024a700) :  INFO (__init__:53) - Getting Search Settings for: Bratty Sis 46308
2019-07-17 20:34:23,478 (14ecf024a700) :  INFO (PAsearchSites:887) - mediaTitle w/ possible abbrieviation: Bratty Sis 46308
2019-07-17 20:34:23,487 (14ecf024a700) :  INFO (PAsearchSites:1055) - mediaTitle w/ possible abbrieviation fixed: Bratty Sis 46308
2019-07-17 20:34:23,492 (14ecf024a700) :  INFO (PAsearchSites:1070) - ^^^^^^^ siteID: 541
2019-07-17 20:34:23,492 (14ecf024a700) :  INFO (PAsearchSites:1072) - ^^^^^^^ Shortening Title
2019-07-17 20:34:23,492 (14ecf024a700) :  INFO (PAsearchSites:1073) - bratty sis vs bratty sis
2019-07-17 20:34:23,493 (14ecf024a700) :  INFO (PAsearchSites:1076) - 1
2019-07-17 20:34:23,493 (14ecf024a700) :  INFO (PAsearchSites:1089) - 6
2019-07-17 20:34:23,493 (14ecf024a700) :  INFO (PAsearchSites:1101) - searchTitle (before date processing): 46308
2019-07-17 20:34:23,493 (14ecf024a700) :  INFO (__init__:63) - Search Title: 46308
2019-07-17 20:34:23,493 (14ecf024a700) :  INFO (__init__:72) - 46308
2019-07-17 20:34:23,498 (14ecf024a700) :  DEBUG (networking:161) - Fetching 'https://brattysis.com/video/watch/46308' from the HTTP cache
2019-07-17 20:34:23,502 (14ecf024a700) :  CRITICAL (agentkit:1014) - Exception in the search function of agent named 'PhoenixAdult', called with keyword arguments {'year': None, 'id': '26198', 'name': 'Bratty Sis 46308'} (most recent call last):
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-22929c8a2/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/api/agentkit.py", line 1007, in _search
    agent.search(*f_args, **f_kwargs)
  File "/config/Library/Application Support/Plex Media Server/Plug-ins/PhoenixAdult.bundle/Contents/Code/__init__.py", line 616, in search
    results = PAsearchSites.networkNubiles.search(results,encodedTitle,title,searchTitle,siteNum,lang,searchByDateActor,searchDate, searchSiteID)
  File "/config/Library/Application Support/Plex Media Server/Plug-ins/PhoenixAdult.bundle/Contents/Code/networkNubiles.py", line 31, in search
    searchResult = searchResults.xpath('//div[@class="descrips"]')[0]
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-22929c8a2/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/code/sandbox.py", line 108, in <lambda>
    _getitem_           = lambda x, y: x.__getitem__(y),
IndexError: list index out of range

2019-07-17 20:34:23,503 (14ecf024a700) :  DEBUG (runtime:88) - Sending packed state data (1016 bytes)
2019-07-17 20:34:23,504 (14ecf024a700) :  DEBUG (runtime:924) - Response: [200] str, 16 bytes
```

**Comment by SgtBatten** [comment]
They started rolling out network wide changes yesterday. working on it

**Comment by SgtBatten** [comment]
Please try #291 

I was unable to test every site. Ran out of time.

**Comment by HWatBobby** [comment]
@SgtBatten 
Looks like it worked. Tested on Bratty Sis, Moms Teach Sex, My Family Pies, PetiteHD, Princess Cum, and Step Siblings Caught. All found the videos in the search with just a couple minor inconveniences. Step Siblings Caught required the search term to be stepsiblingscaught, wouldn't match with spaces, to find a match. All pulled metadata, images and moved scenes to the proper collections except for Bratty Sis. Bratty Sis matches pull Title and Summary only. Have tag manually for collection and poster is just a screen grab from the video.

Other than those little things it's back to pulling data. Thanks for the quick fix!

**Comment by SgtBatten** [comment]
My understanding has always been no spaces. I didn't know it was even possible with spaces so there you go.  My guess is step siblings caught doesn't work due to a conflict with step siblings (different)

My changes didn't alter anything to do with that. It was all just xpath changes. Still some more work required but mostly sorted.

**Comment by SgtBatten** [comment]
I'll look into bratty sis today

**Comment by SgtBatten** [comment]
Bratty Sis is fixed in #291 now

testing other sites

**Comment by HWatBobby** [comment]
It looks like it's working for all the videos I have. No issues with any of the matches. Awesome work!
